frontend/tests/e2e_tests/run.js

```diff
@@ -257,7 +257,7 @@ const composeLogs = async config =>
     '📋 Collecting container logs...',
     async () => {
       console.log('getting em logs');
-      const result = await compose.logs(getComposeOptions(config));
+      const result = await compose.logs([], getComposeOptions(config));
```

Comment:
oh my 🤦 - thanks for catching that, I've seen the corresponding log output, but didn't look into it yet, as I primarily rely on the trace info alone 😬